NitroxPatcher/Patches/Dynamic/ErrorMessage_OnLateUpdate_Patch.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using HarmonyLib;
+using NitroxModel.Helper;
+using NitroxPatcher.Patches;
+using UnityEngine;
+
+/// <summary>
+/// Ensures the in-game log is animated smoothly regardless of the time scale.
+/// </summary>
+public sealed partial class ErrorMessage_OnLateUpdate_Patch : NitroxPatch, IDynamicPatch
```

Comment:
Looking at `GameAnalytics_Patch` this should probably not be marked partial as our code generation doesn't need to generate the Patch method.